dkpro-core-io-nyt-asl/src/test/java/de/tudarmstadt/ukp/dkpro/core/io/nyt/NYTCollectionReaderTest.java

```diff
@@ -0,0 +1,53 @@
+package de.tudarmstadt.ukp.dkpro.core.io.nyt;
+
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+
+import org.apache.uima.UIMAException;
+import org.apache.uima.analysis_engine.AnalysisEngine;
+import org.apache.uima.collection.CollectionReader;
+import org.apache.uima.fit.component.CasDumpWriter;
+import org.apache.uima.fit.factory.AnalysisEngineFactory;
+import org.apache.uima.fit.factory.CollectionReaderFactory;
+import org.apache.uima.fit.factory.TypeSystemDescriptionFactory;
+import org.apache.uima.fit.pipeline.SimplePipeline;
+import org.apache.uima.resource.ResourceInitializationException;
+import org.apache.uima.resource.metadata.TypeSystemDescription;
+import org.junit.Test;
+
+
+public class NYTCollectionReaderTest {
```

Comment:
Tests for IO components should use [testOneWay or testRoundTrip](https://dkpro.github.io/dkpro-core/releases/1.8.0/docs/developer-guide.html#_testing_i_o_componets) which also do a consistency checking of the CAS.